omero/developers/JavaGateway.txt

```diff
@@ -0,0 +1,19 @@
+Java Gateway documentation
```

Comment:
Does this really need a whole new page? Absolutely fine if it does, I'm just wondering if it could just live on https://www.openmicroscopy.org/site/support/omero5.2/developers/Java.html since it is only 4 paragraphs - or are there plans to add more content?